### DISCUSSION TITLE
青铜VIP会员应该能开启10条隧道，实际只能开4条

### DISCUSSION THREAD

**Discussion by DingChang90** [natfrp/wiki#116] [使用问题] [answered]
1.因为开启隧道一直提示「您开启的隧道数量已到达上限, 请关闭其他隧道后再试」，固删掉了所有隧道，重新添加。
2.添加了7条隧道， 一起开启，随机开启了4条，其他隧道依然报「您开启的隧道数量已到达上限, 请关闭其他隧道后再试」。
关闭，重启Frp客户端，随机开启了另外4条隧道，继续报「您开启的隧道数量已到达上限, 请关闭其他隧道后再试」。
Nyat ID：234224
用户名：hongyanV

**Comment by fakeboboliu** [comment, marked as answer]
你好，
经检查日志，我们认为这是您到香港服务器的连接不稳定，反复高速重试使服务器暂时不确定隧道在线状态，从而造成的显示错误。
同时我们注意到您使用的客户端并不受我们支持（[相关说明](https://github.com/natfrp/wiki/blob/da1f08fac22a73dd9b621cb34f0ecb6f8ea84fcb/geek.md?plain=1#L42)）。
请考虑使用我们在网站提供的客户端从而让我们能为你消除此类显示错误，但是归根结底，大陆用户请不要使用境外服务器。
